eth-types/src/lib.rs

```diff
@@ -248,7 +249,7 @@ struct GethExecStepInternal {
 
 /// The execution step type returned by geth RPC debug_trace* methods.
 /// Corresponds to `StructLogRes` in `go-ethereum/internal/ethapi/api.go`.
-#[derive(Clone, Eq, PartialEq)]
+#[derive(Clone, Eq, PartialEq, Serialize)]
```

Comment:
it's for debug purpose. it makes things easier by `printlin!("{}", serde_json::to_string(traces).unwrap())`

Reply:
I want to get a traces json like geth. It's nice to have a json output?

Reply:
> I want to get a traces json like geth. It's nice to have a json output?

That fine!  I was just mentioning the `Debug` format option in case you were not aware of it :)